Adds Fernet, a digestif that reduces your satiety.

ThePainkiller opened the pull request:
:cl: Dimmadunk
add: There's a new type of reagent added to the booze dispenser: Fernet. With it you can make a couple of new digestif drinks that will help you reduce excess fat!
/ :cl:

Have you ever bit more than you could chew and regretted it later?
Have you ever wanted to chew on a Gaia leaf to heal yourself but you can't force any more food down your throat?
Did you ever want to reduce your satiety FAST without poisoning you in the process with lipolicide?
Do you just like getting fucking smashed at the bar with a higher drink repertoire?

Introducing **Fernet**. A world-wide known digestif of italian origin commonly mixed with cola, with three recipes to be made out of it for different palates:

**Fernet Cola:** the classic, helps reducing your hunger

**Fanciulli:** the REAL way to prepare a Manhattan (since it ACTUALLY uses a bitter!), a sobering drink that makes you lazy (causes an initial tick of stamina damage the first time it enters your body)

**Branca Menta:** the refreshing one, perfect for a hot summer afternoon after eating too much meat. It cools you down if overheated, and makes you even more lazy than Fanciulli.

The marvelous spriting work is all done by [Dantebumer](http://www.byond.com/members/Dantebumer).

Values are more than open to change due to balance reasons as mantainers see fit. I do not want to make this drink look like a cheap tirizene.

ThePainkiller committed:
Added Fernet and subdrinks

Added sprites for Fernet (bottle), and glasses of pure Fernet, Fernet Cola, Fanciulli and Branca Menta

ThePainkiller committed:
Adds Fernet and drinks made out of it

Added Fernet, Fernet Cola, Fanciulli and Branca Menta

ThePainkiller committed:
Adds reactions to fernet

ThePainkiller committed:
Adds Bottle of Fernet to the list

Spriting by the wonderful NeoExperiences

ThePainkiller committed:
Adds Fernet to the Borg's Shaker

They can help you indulge in more hedonism now.

ThePainkiller committed:
Adds Fernet to the Booze Dispenser

ThePainkiller committed:
Adds Bottle of Fernet to the list

ThePainkiller committed:
Updated Fanciulli, Branca Menta

Oops, nearly made two of the drinks not cause the satiety reduction.

ThePainkiller committed:
Fixed a missing comma on reagent list

I'm dumb, I forgot this was a list

Wilchenx commented:
Remember to also update our wiki

ThePainkiller commented:
If it gets approved and merged, I will update the wiki as well!

It does need more updating since it doesn't have grenadine, I had to dive on the code to see how a tequila sunrise was made.

ThePainkiller committed:
Fixed the procs on stamina and hunger

I'm sorry I'm sleepy and retarded to mistake an L for an M

RicardoFort commented:
HACEME UN PR
HACEME UN PR
QUE ESTA NOCHE
QUIERO GOZAR

Barhandar commented on code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:
```diff
@@ -1612,4 +1612,93 @@ All effects don't start immediately, but rather get worse over time; the rate is
+	description = "What if the Manhattan coctail ACTUALLY used a bitter herb liquour? Helps you sobers up." //also causes a bit of stamina damage to symbolize the afterdrink lazyness
+	color = "#CA933F" // rgb: 202, 147, 63
+	boozepwr = -10
+	taste_description = "a sweet sobering mix"
```
Does taste system not handle the a/an particle by itself?

ThePainkiller committed:
Changes the recipe of Fanciulli to include Manhattan

They were conflicting, and the description of the glass already says it's just Manhattan with Fernet, so it's consistent.

ExcessiveUseOfCobblestone commented:
Why is furnetpuro a sprite? looks like the normal drinking glass?

ThePainkiller commented:
Let's play the _"how many layers of sleep deprivation are you on while coding"_ game
Excuse me for being this dumb and keep correcting my own messes

ThePainkiller commented:
@ExcessiveUseOfCobblestone It's a drinking glass filled with a certain gradient of syrupy, very dark green
We wanted it to NOT look like the Irish Car Bomb, but still make it look different than a drinking glass filled with Charcoal.

ExcessiveUseOfCobblestone commented:
@ThePainkiller I haven't looked at these sprites in 5ever since I deleted half of them for the exact same reason but if you can do it through reagent color please do that over new sprite.

ThePainkiller commented:
@ExcessiveUseOfCobblestone that would require to just remove the glass_icon_state line, wouldn't it? It would become just a reagent inside a glass with its hex value showing as liquid.

ExcessiveUseOfCobblestone commented:
Yep! I tried to make it simple when I coded that sucker 200000 years ago

ExcessiveUseOfCobblestone commented:
Also if you plan on making more cool features please hop on #coderbus (irc) or Coding-General (discord).

I see you are making edits on master and it becomes a real pain real fast. Don't want you to get discouraged because of the funs of git lol.

ThePainkiller committed:
Removes glass_icon_state from pure fernet

Cobby said it was pointless since the spritework was lazy

ThePainkiller commented:
Sorry, I'm still trying to get the gist of github and PR's, other than fixing grammar mistakes here and there, but then again I blame my sleep deprivation and fatigue
This is truly my first huge pr editing multiple files (and actually adding content)

Tlaltecuhtli commented:
use the web editor its gr8
when looking at a file u press edit (make sure u on master branch, switch it in the top left of the file)
change the line and then merge it into thr branch used for this pr

Iamgoofball commented:
This is cool but we do already have a weight loss reagent. Lipolicide.

PKPenguin321 commented:
That is not how you make a changelog, I don't think that will actually work at all

davethwave commented:
> This is cool but we do already have a weight loss reagent. Lipolicide.

Lipolicide is a toxin that deals toxin damage and probably more effective and dangerous, while this is made for casual use and doesn't sap your hunger as much per unit.

juanototo2 commented:
![image](https://user-images.githubusercontent.com/21270901/40139556-a1296e4e-5926-11e8-9b32-8733b31097c8.png)

ExcessiveUseOfCobblestone commented:
This also shifts the downside to potential liver damage which is a lot more longterm than just toxin.

ThePainkiller commented:
@ExcessiveUseOfCobblestone The thing is that it's supposed to not be binge-drank, just take a bit, reduce your satiety, wait out the stamina loss and be on your way. I'd say it's a decent trade off for lipolicide being harder to get (either by hacking botany and actually bothering to plant cannabis, or combining diethylamine ephedrine and whatever else made it)
On a side note, one contains negative boozepwr (since it's a sobering coctail) and the fernet cola has the same boozepwr value as beer (25), wich is on the second lowest tier of alcoholic power.
As I said, I'm always open for reccomendations on how to balance it out to make it coherent!

@PKPenguin321 I apologize, I am still learning the ropes on how to format my PR's and I was told to do the cl slash cl thing, but they never bothered explaining me how the :cl: works, or why the fuck do I need to format it like that. I will fix it to better format it to be changelog compliant. Something like:

:cl: Dimmadunk
add: There's a new type of reagent added to the booze dispenser: Fernet. With it you can make a couple of new digestif drinks that will help you reduce excess fat!
/ :cl:

Will that work?

Barhandar commented:
Yes, but formatting and its workings are obvious from how it's formatted in the automatic example on opening the PR. Unless you did that somehow bypassing the text.

ThePainkiller commented:
Time to edit the wiki it seems! <3

Aranclanos commented:
based
